src/core/include_urls.py

```diff
@@ -189,6 +190,8 @@
     url(r'^download/file/(?P<file_id>\d+)/$', journal_views.download_journal_file, name='journal_file'),
 
     url(r'^set-timezone/$', core_views.set_session_timezone, name='set_timezone'),
+
+    url(r'^jsi18n/$', JavaScriptCatalog.as_view(), name='javascript-catalog'),
```

Comment:
Sounds like we should cache this view, since it will parse the translation files on every request.